### ISSUE TITLE
Unable to create a pin with websites without picture

### ISSUE THREAD

**Issue by rcampusa** [pinry/pinry#299]
I dont know if this is a feature request or is a bug. 

I am unable to add a pin to websites that doesnt have a picture, even if I add a picture myself. This website is an example:

https://www.sixdegreesofwikipedia.com/

Is this a bug, or a feature request?

**Comment by winkidney** [comment]
It's a feature request, we don't support to add a website link as Pin currently.

**Comment by cleaverm** [comment]
> even if I add a picture myself.

Is this just an issue with the bookmarklet or maybe the browser extensions? I could confirm that the bookmarklet (still) doesn't work for https://www.sixdegreesofwikipedia.com/, but choosing "Create -> Pin" in the Pinly UI let me upload a local image and save a pin for www.sixdegreesofwikipedia.com.

So while there seems to be room for improvement, there is a workaround.

**Comment by winkidney** [comment]
Seems bookmarklet work but no image detected in that page.
Does your Pinry instance has been deployed with HTTPS support?

![image](https://user-images.githubusercontent.com/4109722/163661909-e096946d-5cbf-4635-a5bd-893c5e9629ed.png)

**Comment by cleaverm** [comment]
Yeah, the bookmarklet would need to be updated to provide some sort of UI for pages that have no image (or I suppose to just skip the image selection and go straight to the add pin action).

My instance (which I tested with) is using HTTPS.
